### PR TITLE
docs overview page, reworked intro page

### PR DESCRIPTION
**Pull request by romain-grecourt**
Add docs/src/main/docs/about/01_overview.adoc
Reworked docs/src/main/docs/about/02_introduction.adoc

update sitegen to 1.0.8
renamed docs/src/main/docs/tracing/01_tracing.adoc to lowercase only
renamed docs/src/main/docs/tracing/02_zipkin.adoc to lowercase only
removed empty line in docs/src/main/docs/guides/01_overview.adoc

### CODE REVIEW

**Comment by romain-grecourt** [comment]
@kumar-dhanagopal Can you review this please ?

File: docs/src/main/docs/about/01_overview.adoc [hunk 1]
```diff
@@ -0,0 +1,103 @@
+///////////////////////////////////////////////////////////////////////////////
+
+    Copyright (c) 2019 Oracle and/or its affiliates. All rights reserved.
```
Comment (kumar-dhanagopal) [inline docs/src/main/docs/about/01_overview.adoc]: [nit, missing comma]

Copyright (c) 2019, Oracle and/or its affiliates. All rights reserved.

**Comment by kumar-dhanagopal** [comment]
@romain-grecourt - It appears from your earlier comment that only docs/src/main/docs/about/02_introduction.adoc has changes that I should review. Pls confirm.

File: docs/src/main/docs/about/01_overview.adoc (hunk 1 above)
Comment (romain-grecourt) [reply docs/src/main/docs/about/01_overview.adoc]: We actually don't put a comma when there is a single year, see the template at etc/copyright.txt

File: docs/src/main/docs/about/02_introduction.adoc [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+:description: about Helidon
+:keywords: helidon, java, microservices, microprofile
+
+== A collection of Java libraries
+
+Helidon is a collection of Java libraries for writing microservices.
+
+There is no specific tooling or deployment model, you simply use your favorite
+ tools !
+
```
Comment (kumar-dhanagopal) [inline docs/src/main/docs/about/02_introduction.adoc]: There's no need for any specific tooling or deployment model. Simply use your favorite tools !

File: docs/src/main/docs/about/02_introduction.adoc [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+NOTE: Your microservice is a plain Java application, you do not need an
+ application server.
```
Comment (kumar-dhanagopal) [inline docs/src/main/docs/about/02_introduction.adoc]: Make "you do not need..." a separate sentence.

File: docs/src/main/docs/about/02_introduction.adoc [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
+== Cloud native Java
```
Comment (kumar-dhanagopal) [inline docs/src/main/docs/about/02_introduction.adoc]: Cloud-native Java

**Comment by romain-grecourt** [comment]
> @romain-grecourt - It appears from your earlier comment that only docs/src/main/docs/about/02_introduction.adoc has changes that I should review. Pls confirm.

Both should be reviewed, some of the text in the new overview page is re-used and modified some is new.

File: docs/src/main/docs/about/02_introduction.adoc [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+== Helidon MP
+
+Helidon MP is the branded name for our https://microprofile.io[MicroProfile]
+ implementation.
+
+https://microprofile.io[MicroProfile] is a platform definition that is familiar
+ to enterprise Java developers, it provides a declarative programming style that
+ is portable. It includes popular Java APIs such as `JAX-RS` and `CDI`.
+
+NOTE: Helidon MP is built on top the Helidon libraries.
```
Comment (kumar-dhanagopal) [inline docs/src/main/docs/about/02_introduction.adoc]: NOTE: Helidon MP is built on top of the Helidon libraries.

File: docs/src/main/docs/about/02_introduction.adoc [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+NOTE: Helidon SE gives you full transparency and puts you in control. There is
+ no magic!
+
+== Helidon MP
+
+Helidon MP is the branded name for our https://microprofile.io[MicroProfile]
+ implementation.
+
+https://microprofile.io[MicroProfile] is a platform definition that is familiar
+ to enterprise Java developers, it provides a declarative programming style that
```
Comment (kumar-dhanagopal) [inline docs/src/main/docs/about/02_introduction.adoc]: Make "it provides..." a separate sentence.

File: docs/src/main/docs/about/02_introduction.adoc [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+* https://prometheus.io/[Prometheus]
+* https://opentracing.io/[OpenTracing]
+* https://coreos.com/etcd/[Etcd]
+
+TIP: The <<getting-started/02_base-example.adoc,Helidon Quickstart Examples>>
+ contain support for Docker and Kubernetes.
+
+== Helidon SE
+
+Helidon SE is the branded name for the functional programming style provided
```
Comment (kumar-dhanagopal) [inline docs/src/main/docs/about/02_introduction.adoc]: "branded name"?  I'm not sure what you intended.

I suggest: "Helidon SE supports the functional programming style."

File: docs/src/main/docs/about/02_introduction.adoc [hunk 8]
```diff
@@ -0,0 +1,63 @@
...
+ by all Helidon libraries.
+
+NOTE: Helidon SE gives you full transparency and puts you in control. There is
+ no magic!
+
+== Helidon MP
+
+Helidon MP is the branded name for our https://microprofile.io[MicroProfile]
+ implementation.
+
```
Comment (kumar-dhanagopal) [inline docs/src/main/docs/about/02_introduction.adoc]: Again, not sure what "branded name" means.

I suggest: "Helidon MP is our https://microprofile.io[MicroProfile] implementation.

File: docs/src/main/docs/about/02_introduction.adoc [hunk 9]
```diff
@@ -0,0 +1,63 @@
...
+https://microprofile.io[MicroProfile] is a platform definition that is familiar
```
Comment (kumar-dhanagopal) [inline docs/src/main/docs/about/02_introduction.adoc]: There's no need to provide a link again considering there's a link in just the previous sentence. Alternatively, keep the link here and remove it from the previous sentence.

File: docs/src/main/docs/about/01_overview.adoc [hunk 10]
```diff
@@ -0,0 +1,103 @@
...
+
+== Get going
+
+[PILLARS]
+====
+[CARD]
+.Getting Started
+[icon=assistant,link=getting-started/02_base-example.adoc]
+--
+Quickly create your first application under 2 minutes.
```
Comment (kumar-dhanagopal) [inline docs/src/main/docs/about/01_overview.adoc]: Quickly create your first application, in under 2 minutes.

File: docs/src/main/docs/about/01_overview.adoc [hunk 11]
```diff
@@ -0,0 +1,103 @@
...
+--
+
+[CARD]
+.Guides
+[icon=explore,link=guides/01_overview.adoc]
+--
+Hands-on guides.
+--
```
Comment (kumar-dhanagopal) [inline docs/src/main/docs/about/01_overview.adoc]: Follow step-by-step guides to build your applications.

File: docs/src/main/docs/about/01_overview.adoc [hunk 12]
```diff
@@ -0,0 +1,103 @@
...
+
+[CARD]
+.Security
+[icon=security,link=security/01_introduction.adoc]
+--
+A tool-chain to handle authentication, authorization and context propagation.
+--
+
+[CARD]
+.Microprofile
```
Comment (kumar-dhanagopal) [inline docs/src/main/docs/about/01_overview.adoc]: .MicroProfile

File: docs/src/main/docs/about/01_overview.adoc [hunk 13]
```diff
@@ -0,0 +1,103 @@
...
+[icon=widgets,link=microprofile/01_introduction.adoc]
+--
+MicroProfile is a platform definition that is familiar to enterprise Java
+ developers.
+--
```
Comment (kumar-dhanagopal) [inline docs/src/main/docs/about/01_overview.adoc]: A platform definition that is familiar to enterprise Java developers.

(for consistency with the other descriptions)

File: docs/src/main/docs/about/01_overview.adoc [hunk 14]
```diff
@@ -0,0 +1,103 @@
...
+
+[CARD]
+.Extensions
+[icon=extension,link=extensions/01_overview.adoc]
+--
+A set of add-ons for your Helidon based applications.
```
Comment (kumar-dhanagopal) [inline docs/src/main/docs/about/01_overview.adoc]: A set of add-ons for your Helidon-based applications.

**Comment by kumar-dhanagopal** [comment]
I reviewed docs/src/main/docs/about/01_overview.adoc and docs/src/main/docs/about/02_introduction.adoc, and posted my feedback.  Let me know if I should review any other doc.

File: docs/src/main/docs/about/02_introduction.adoc (hunk 7 above)
Comment (romain-grecourt) [reply docs/src/main/docs/about/02_introduction.adoc]: The intent is to describe SE, it is referred to is many places but not explicitly described.

How about "Helidon SE is the functional programming style" ?

Comment (kumar-dhanagopal) [reply docs/src/main/docs/about/02_introduction.adoc]: Sounds good.

**Comment by romain-grecourt** [comment]
Thanks, I've incorporated all reviews. Will merge when the pipeline passes.

**Pull request merged**